ui/tests/acceptance/secrets/backend/kv/secret-test.js

```diff
@@ -146,7 +147,8 @@ module('Acceptance | secrets/secret/create, read, delete', function (hooks) {
     hooks.afterEach(async function () {
       await consoleComponent.runCommands([`delete sys/mounts/${this.backend}`]);
     });
-    test('version 1 performs the correct capabilities lookup and does not show metadata tab', async function (assert) {
+    skip('version 1 performs the correct capabilities lookup and does not show metadata tab', async function (assert) {
```

Comment:
I'd argue it's still asserting that capabilities are correctly being asserted for v1. The metadata piece can probably be removed but isn't harmful

Reply:
Good point. i'll amend and take out the metadata part.